functions/services/bls_service.py

```
@@ -0,0 +1,657 @@
+    Args:
+        zip_code: 5-digit US zip code
+        trade: Trade name (e.g., "electrician")
+
+    Returns:
+        BLSLaborRate for the specified trade
+
+    Raises:
+        ValueError: If trade is not recognized
+    """
+    if trade not in SOC_CODE_MAP:
+        raise ValueError(f"Unknown trade: {trade}. Valid trades: {list(SOC_CODE_MAP.keys())}")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)